src/parser.js

```diff
@@ -798,25 +798,83 @@ export default class Parser {
       Array: ArrayElement, Boolean: BooleanElement, Number: NumberElement,
       String: StringElement,
     } = this.minim.elements;
+
     let element;
+    let Type;
 
     // Convert from Swagger types to Minim elements
     if (parameter.type === 'string') {
-      element = new StringElement('');
+      Type = StringElement;
     } else if (parameter.type === 'integer' || parameter.type === 'number') {
-      element = new NumberElement();
+      Type = NumberElement;
     } else if (parameter.type === 'boolean') {
-      element = new BooleanElement();
+      Type = BooleanElement;
     } else if (parameter.type === 'array') {
+      Type = ArrayElement;
+    } else {
+      // Default to a string in case we get a type we haven't seen
+      Type = StringElement;
+    }
+
+    if (parameter.enum) {
       element = new ArrayElement();
+      element.element = 'enum';
 
-      if (parameter.items) {
-        element.content = [this.convertParameterToElement(
-          parameter.items, (path || []).concat(['items']), true)];
-      }
+      _.each(parameter.enum, (value, index) => {
```

Comment:
if `parameter.enum` is array, then you can use built-in `parameter.enum.forEach((value, index) => {`